### PR TITLE
BNGP-3953: Update 'Check and submit' page to reflect v21 of the prototype

### PR DESCRIPTION
**Pull request by usman-j-shaikh**

### CODE REVIEW

File: packages/webapp/client/sass/application.scss [hunk 1]
```diff
@@ -3,7 +3,7 @@ $govuk-assets-path: '/public/';
 $govuk-images-path: '#{$govuk-assets-path}images/';
 $govuk-fonts-path: '#{$govuk-assets-path}fonts/';
 $govuk-global-styles: true;
-$govuk-font-family: "Arial", sans-serif !important;
+$govuk-font-family: "GDS Transport", arial, sans-serif !important;
```
Comment (paulsimonandrews) [inline packages/webapp/client/sass/application.scss:6]: Remove "GDS Transport", we can't use it for licensing reasons. Correct font throughout should be arial

File: packages/webapp/src/views/land/check-and-submit.html [hunk 2]
```diff
@@ -3,219 +3,217 @@
...
+      {% include "common/applicant-information.html" %}
 
-        <!--Land boundary-->
-        <h2 class="govuk-heading-l">Land boundary</h2>
-        
-        {% set landBoundaryUploadTypeElement = {
-          key: {
-              text: "Land boundary file type"
+      <!--Land boundary-->
+      <h2 class="govuk-heading-m">Land boundary</h2>
```
Comment (paulsimonandrews) [inline packages/webapp/src/views/land/check-and-submit.html:20]: Wording doesn't match the copy deck

File: packages/webapp/src/views/land/check-and-submit.html [hunk 3]
```diff
@@ -3,219 +3,217 @@
...
-          key: {
-            text: "Land boundary on a map"
-          },
-          classes: hideClass(documentOrImage) + ' defra-js-show hidden',
-          value: {
-            html: "<div id=\"map\" data-testid=\"land-boundary-map-value\" class=\"map map--location\" tabindex=\"0\"></div>"
-          }
-        } if ENABLE_ROUTE_SUPPORT_FOR_GEOSPATIAL === 'Y' else undefined %}
+      <!--Land ownership-->
+      <h2 class="govuk-heading-m">Land ownership</h2>
```
Comment (paulsimonandrews) [inline packages/webapp/src/views/land/check-and-submit.html:115]: Doesn't match copy deck. This section should come after applicant information

File: packages/webapp/src/views/land/check-and-submit.html [hunk 4]
```diff
@@ -3,219 +3,217 @@
...
+        }
+      } if ENABLE_ROUTE_SUPPORT_FOR_GEOSPATIAL === 'Y' else undefined %}
+
+      {{ govukSummaryList({
+        classes: "govuk-!-margin-bottom-9",
+        rows: [
+          landBoundaryUploadTypeElement,
+          {
+            key: {
+              text: "Land boundary file uploaded"
```
Comment (paulsimonandrews) [inline packages/webapp/src/views/land/check-and-submit.html:59]: Wording doesn't match copy deck

**Review by paulsimonandrews — CHANGES_REQUESTED** [review]
Changes don't fully match prototype/copydeck

**Review by pwadmore-ea — CHANGES_REQUESTED** [review]
image

- This does not reflect the copydeck/other uploads types . Other upload types group multiple files in a single row such as.

image

- If other landholders or leaseholders are present they should be output. This does not match the copydeck:

image

Compare with copydeck:
image

**Comment by usman-j-shaikh** [comment]
> image
> 
> * This does not reflect the copydeck/other uploads types . Other upload types group multiple files in a single row such as.
> 
> image
> 
> * If other landholders or leaseholders are present they should be output. This does not match the copydeck:
> 
> image
> 
> Compare with copydeck: image

@pwadmore-ea 
Land Ownership issue already covered in https://github.com/DEFRA/biodiversity-net-gain-service/pull/587

**Review by pwadmore-ea — APPROVED** [review]

**Review by paulsimonandrews — APPROVED** [review]

**Comment by shaundefra** [comment]
tested locally

**Pull request merged**